integration-tests/test_satellite.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+import subprocess
+import os
+import yaml
+import traceback
+import contextlib
+from time import sleep
+import re
+
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+def test_obfuscation(insights_client, tmp_path):
+    """
+    Test if the satellite hostname is obfuscated
+    Related to https://bugzilla.redhat.com/show_bug.cgi?id=1963917
+
+    """
+    # Register system to insights
+    insights_client.register()
+    # assert insights_client.is_registered
+
+    # Record the current hostname
+    with open("/etc/hostname", "r") as f:
+        previous_hostname = f.read().strip()
+
+    try:
+        # Set a new hostname
+        hostname_to_test = "insights-client-satellite"
+        set_hostname(hostname_to_test)
+        cmd = ["hostnamectl", "status"]
+        output = subprocess.check_output(cmd, universal_newlines=True)
+        assert hostname_to_test in output
+
+        # redaction on hostname:
+        data_redaction(hostname_to_test)
+
+        # check the collection, if redaction pattern is filtered out
+        # collect data
+        archive_result = insights_client.run("--no-upload")
+
+        # extract the collected tar file
+        m = re.search("^Archive saved at (.+)$", archive_result.stdout, re.MULTILINE)
+        assert m
+        archive_location = m.group(1)
+        cmd = "tar -xzvf %s -C %s" % (archive_location, tmp_path)
+        subprocess.check_call(cmd, shell=True)
+
+        # the redaction data should not locate in the collected files
+        archive_extract = archive_location.split(".")[0].split("/")[-1]
+        path_to_check = tmp_path / archive_extract
+        cmd = "grep -rnw . -e %s" % hostname_to_test
+        # should not find the obfuscate pattern, the return code should not zero.
+        ret = subprocess.run(cmd, shell=True, cwd=path_to_check)
+        assert ret.returncode == 1
+    finally:
+        # Recover the previous hostname for the system
+        set_hostname(previous_hostname)
+        # Remove all data redaction
+        os.remove("/etc/insights-client/file-content-redaction.yaml")
```

Comment:
since this path is used in a couple of places, it makes sense to factorize it into a module level variable: for example, add
```python
CONTENT_REDACTION_FILE = "/etc/insights-client/file-content-redaction.yaml"
```
before the tests (right after the `import`s), and use it in the two places